tests/source/expr.rs

```diff
@@ -79,7 +79,7 @@ fn bar() {
     let bar = 5 ;
     let nonsense = (10 .. 0)..(0..10);
 
-    loop{if true {break}}
+    loop{if true {break;}}
```

Comment:
I added the semicolon because cargo test failed without it. The target file expected it to be formatted on 3 lines and it keeps it to 1 since a statement without a semicolon. Turning this pr into a config option will remove this, so I'll do that.

Reply:
Thanks for the explanation!